mysql-test/suite/rocksdb/t/persistent_cache.test

```diff
@@ -0,0 +1,40 @@
+--source include/have_rocksdb.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+--enable_warnings
+
+--let $_server_id= `SELECT @@server_id`
+--let $_expect_file_name= $MYSQLTEST_VARDIR/tmp/mysqld.$_server_id.expect
+--exec echo "wait" >$_expect_file_name
+
+# restart server with correct parameters
+shutdown_server 10;
+--exec echo "restart:--rocksdb_persistent_cache_path=/tmp/persistent_cache --rocksdb_persistent_cache_size=1000000000" >$_expect_file_name
```

Comment:
This should use the $MYSQL_VARDIR path instead of /tmp. Otherwise, running tests in parallel could cause conflict with /tmp/persistent_cache.